superset-frontend/package-lock.json

```diff
@@ -21363,28 +21363,28 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "resolved": false,
+              "resolved": "",
```

Comment:
I've been having issues with package-lock doing this too. How do we stop the flip-floping here!? Wondering if it's disagreeing NPM version(?)

Reply:
I wonder whether it would help if we lock npm version in `package.json` (via [engines](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines) config) and add a `.npmrc` file.

Reply:
seems like a good idea to me